extensions/amqp/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-build-parent</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../../poms/build-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-amqp-parent</artifactId>
+    <name>Camel Quarkus :: AMQP</name>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>deployment</module>
+        <module>runtime</module>
+    </modules>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom-deployment</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.amqphub.quarkus</groupId>
+                <artifactId>quarkus-qpid-jms-deployment</artifactId>
+                <version>${quarkus.qpid.jms.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!-- quarkus-qpid-jms-deployment is currently not part quarkus-qpid-jms-deployment-->
+            <dependency>
+                <groupId>org.amqphub.quarkus</groupId>
+                <artifactId>quarkus-qpid-jms-deployment</artifactId>
+                <version>${quarkus.qpid.jms.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
I'd expect these to be in the deployment module. Is there any reason for them to be here in the parent?

Reply:
By importing the deployment BOMs here in the parent, we awake an impression that it is legal to use the deployment artifacts in all descendants, incl. the runtime module, which I think really should not happen.

Reply:
I'll fix this.

I also noticed some other extensions where this happens. I'll follow up with another PR to fix tomorrow.

Reply:
I have some dep. management cleanup in progress. I'd fix all these things there, if you do not mind.